zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/inbound.kt

```diff
@@ -58,14 +58,39 @@ internal interface InboundCallHandler {
  */
 @PublishedApi
 internal class InboundCall(
-  private val context: InboundBridge.Context,
-  val funName: String,
   val encodedArguments: Array<String>,
 ) {
+  internal lateinit var context: InboundBridge.Context
+  internal val serviceName: String
+  internal val funName: String
+  internal val callbackName: String
+
   private val arguments = ArrayList<Any?>(encodedArguments.size / 2)
   private var callStartResult: Any? = null
   private var i = 0
 
+  init {
```

Comment:
This is where all the other deleted code went! Now encodedArguments does more stuff.